### ISSUE TITLE
Allow setting up color format

### ISSUE THREAD

**Issue by vitvakatu** [three-rs/three#109]
By default we have Srba8 color format, we should provide ability to change it to Rbga8 on demand.

**Comment by kvark** [comment]
We still use Srgba8 for textures, don't we? That still works? Hmm...
If only framebuffer is unsupported, I think we should render to an off-screen target and then blit to the final target with gamme correction done in the shader. That may work well as a case for the post-fx pipeline.

**Comment by vitvakatu** [comment]
Yes, textures are still in `Srbg`, changing `ColorFormat` in `render` module won't affect this.
Am I understood correctly that we can use our `decode_color` function (but in shader) to convert from srgb back buffer to linear frame buffer?

**Comment by norru** [comment]
Related: https://github.com/tomaka/glutin/issues/913#issuecomment-335384646

I have worked this around using a Rgba target instead of a Srgba one and then doing the gamma correction in the resolver shader after the tonemapping.
